frontend/src/app.js

```diff
@@ -343,6 +343,7 @@ class App extends React.Component {
                 <TextListItem component="dd"><a href="https://docs.ibutsu-project.org/" target="_blank" rel="noopener noreferrer">docs.ibutsu-project.org</a></TextListItem>
                 <TextListItem component="dt">Report bugs</TextListItem>
                 <TextListItem component="dd"><a href="https://github.com/ibutsu/ibutsu-server/issues/new" target="_blank" rel="noopener noreferrer">Submit an upstream issue</a></TextListItem>
+                <TextListItem component="dd"><a href="https://issues.redhat.com/secure/CreateIssue!default.jspa" target="_blank" rel="noopener noreferrer">Submit a JIRA issue</a></TextListItem>
```

Comment:
Is this correct?

`CreateIssue!default.jspa`

Reply:
@rsnyman It works for me -- takes me to this page:

![Screenshot from 2020-08-04 16-19-18](https://user-images.githubusercontent.com/44065123/89340783-46281500-d66e-11ea-88ee-cff8d62967f1.png)

Reply:
Ah, but the project is set by default -- let me see if there's a way to specify that

Reply:
I guess it should be https://issues.redhat.com/secure/CreateIssue.jspa?pid=12324624&issuetype=1